turftopic/feature_importance.py

```diff
@@ -1,14 +1,32 @@
 import numpy as np
 import scipy.sparse as spr
 from sklearn.metrics import pairwise_distances
-from sklearn.preprocessing import normalize
 
 
 def cluster_centroid_distance(
     cluster_centroids: np.ndarray,
     vocab_embeddings: np.ndarray,
     metric="cosine",
 ) -> np.ndarray:
+    """Computes feature importances based on distances between
```

Comment:
I didn't write a docstring, since this is not exposed in the docs. Cool of you to do it though, thanks :D